charts/linkerd2/templates/heartbeat.yaml

```diff
@@ -17,7 +17,12 @@ metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
 spec:
+  concurrencyPolicy: Replace
+  {{ if .Values.heartbeatSchedule -}}
```

Comment:
Correct me if I'm wrong, commenting out `heartbeatSchedule` makes it so that the value is no longer configurable. If it isn't should this `if` statement be removed since the value will always be the default schedule in the `else` block?

Reply:
I commented it out so that its default is not midnight UTC. But people can still uncomment it and put whatever schedule they want.